services/app-web/src/pages/StateSubmission/RateDetails/SingleRateCert.tsx

```diff
@@ -216,13 +216,6 @@ export const SingleRateCert = ({
                         scanFile={(key) =>
                             handleScanFile(key, 'HEALTH_PLAN_DOCS')
                         }
-                        deleteFile={(key) =>
-                            handleDeleteFile(
-                                key,
-                                'HEALTH_PLAN_DOCS',
-                                previousDocuments
```

Comment:
One more component to remove `previousDocuments` and `parentSubmissionID` from props and `SingleRateCertProps` type.